### ISSUE TITLE
Highland pattern for Kefir.combine

### ISSUE THREAD

**Issue by monfera** [caolan/highland#262]
What's the solution for creating a stream that observes multiple streams, and when a new value arrives from one of the inputs, it recalculates its argument function (using all the arguments from the input streams) and emits its result? Better yet, do it in a way that one set of input streams trigger a recalculation (and emitting of a new value), while the other set of input streams are just being read for their current (last) value when a recalculation is triggered, but a new event in this set of streams doesn't trigger a recalculation. 

In short, what's the corresponding Highland version of this Kefir.js function https://pozadi.github.io/kefir/#combine

**Comment by vqvu** [comment]
You'll need to write a new transform. Something similar to the
implementation of `zipAll`. Use `pull` to get data as needed and emit when
you have enough. This stream will emit arrays, which you can them `map`
over if you'd like.

At the same time, you can choose to emit or not depending on whether or not
the stream that just emitted a value is an active stream.

Sorry if this is vague. I'll try to put something together when I have a
little bit more time.

**Comment by monfera** [comment]
Thank you, I'll take a look!
